zinc/src/sbt-test/source-dependencies/check-recompilations/test

```diff
@@ -1,9 +1,45 @@
+# this was the first checkRecompilations test ever written. it's a good
+# one for understanding how checkDependencies and checkRecompilations
+# are used.
+
+# the first compile command causes round 0 to happen
+
 > compile
 
+# every class gets compiled.
+
+> checkRecompilations 0 A B C D
+
+# now zinc knows the dependency relations between the classes:
+# ┌─┐┌─┐
+# │D││B│
+# └┬┘└┬┘
+# ┌▽┐ │
+# │C│ │
+# └┬┘ │
+# ┌▽──▽┐
+# │A   │
+# └────┘
```

Comment:
😍

Reply:
I already forgot which tool I used to make this :-)